### DISCUSSION TITLE
Хотелки

### DISCUSSION THREAD

**Discussion by RX-ON** [12-venice/battleship#116] [Ideas]
- мины, они просвечиваются как корабли, при попадании отнимается ХП у игрока

- ХП игрока складывается из палуб его кораблей (4+3+3+2+2+2+1+1+1+1=20)

- тучи блокируют видимость. Игрок может стрелять в тучи, но не увидит результат, пока они не развеятся. если в тучах будет мина, то игрок потеряет -1 хп, при этом туча останется на месте. Тучи могут заспавниться в начале раунда в области 3х3, а также могут быть поставлены игроком в процессе игры, тоже 3х3. (Время жизни тучек - 10 ходов или на всю игру, пока не знаю)

- просветка работает 3х3, светит любой объект (мина или корабль) одним маркером.

- тучи МОЖНО просветить

- картечь стреляет по области 3х3

- воздушный шар сбрасывает бомбы (какое-то количество) в выбранную четверть (5х5) рандомной областью

- за каждый ход игрок получает очки (копит энергию), которую тратит на плюшки

- щит, который позволяет отразить один удар, при этом данный квадратик становится виден противнику. Следующим ударом он сможет поразить цель в этом квадрате.

- корабль "подстава" с 2мя палубами - его можно купить в магазине, используется как мишень. Противник узнает о том, что это "муляж" после того, как полностью его потопит.

- право второго выстрела (покупается перед ходом) - это своего рода страховка, если  в первый раз промахнулся - можешь попробовать ещё.

- право переставить свой корабль - можно изменить расположение целого корабля, но только в нетронутые ячейки

- замена своего корабля на муляж с последующей перестановкой. (когда противник атаковал ваш 2х, 3х или 4х палубник, вы можете купить муляж и поставить его на место своего корабля

**Comment by RX-ON** [comment]
- возможность разогнать тучи на поле противника в области 3х3

**Comment by RX-ON** [comment]
- На карте будет спавниться от 1 до 2 островов 1х1. На них будет стоять сундук с золотом. Если противник попадет в этот остров, то он получит очки/энергию/монеты.

**Reply by RX-ON** [reply]
или бочка

**Comment by drzoiberg2005** [comment]
Подумать над механикой попадания в мину=> при попадании в мину отбирать одну палубу у рандомного корабля игрока

**Comment by drzoiberg2005** [comment]
Ремонтный ключ => отремонтировать рандомную палубу

**Comment by drzoiberg2005** [comment]
Смена погоды во время игры=> обнуление карты и перевыставление игроками своих уцелевших своих кораблей
